### DISCUSSION TITLE
20.2.3

### DISCUSSION THREAD

**Discussion by davidjgraph** [jgraph/drawio-desktop#1081] [General]
## Releases Notes for 20.2.3
[Windows Installer](https://github.com/jgraph/drawio-desktop/releases/download/v20.2.3/draw.io-20.2.3-windows-installer.exe)
[Windows No Installer](https://github.com/jgraph/drawio-desktop/releases/download/v20.2.3/draw.io-20.2.3-windows-no-installer.exe)
[macOS - Universal](https://github.com/jgraph/drawio-desktop/releases/download/v20.2.3/draw.io-universal-20.2.3.dmg)
Linux - [deb](https://github.com/jgraph/drawio-desktop/releases/download/v20.2.3/drawio-amd64-20.2.3.deb), [snap](https://snapcraft.io/drawio), [AppImage](https://github.com/jgraph/drawio-desktop/releases/download/v20.2.3/drawio-x86_64-20.2.3.AppImage) or [rpm](https://github.com/jgraph/drawio-desktop/releases/download/v20.2.3/drawio-x86_64-20.2.3.rpm)
[Google Chrome OS](https://chrome.google.com/webstore/detail/drawio-desktop/pebppomjfocnoigkeepgbmcifnnlndla/reviews)

Windows intel x32 releases are marked -ia32-

**ChangeLog:**

- Updates to [draw.io core 20.2.3](https://github.com/jgraph/drawio/blob/v20.2.3/ChangeLog). All changes from 19.0.4 are added in this build.

<hr /><em>This discussion was created from the release <a href='https://github.com/jgraph/drawio-desktop/releases/tag/v20.2.3'>20.2.3</a>.</em>

**Comment by itsayooluwa** [comment]
[diagrams.net](https://app.diagrams.net/) has not been working for days now 

Please look into it ASAP, Thanks 

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/75017703/186979522-d2a1bef2-5681-4aba-af74-a7a262ca3579.png">

**Reply by itsayooluwa** [reply]
It's just stuck on this page for hours
